hikari/webhooks.py

```diff
@@ -126,8 +126,8 @@ async def execute(
             interaction webhooks.
 
             Additionally, flags this can only be set for interaction webhooks
```

Comment:
Given the below comment, does "flags can only be set for interaction webhooks" apply anymore?